### PR TITLE
Create AzureDevOps-Windows.yaml

### PR DESCRIPTION
**Pull request by lukebrogan-mend**
Add azure devops ucli example with windows runner.

### CODE REVIEW

File: CI-CD/AzureDevOps/Mend CLI/AzureDevOps-windows.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      script: |
+	### Build the application with your required package manager, e.g: ###
+    # - task: NodeTool@0
+    # - task: Maven@0
+    # - task: NuGetCommand@0
+    # - task: Gradle@0
+    # - task: PythonScript@0
+    ### Download the Mend Unified CLI ###
+    echo Downloading Mend CLI
+        Invoke-WebRequest -Uri "https://downloads.mend.io/production/unified/latest/linux_amd64/mend" -OutFile "C:\mend\cli\mend.exe"
```
Comment (tabacmend) [inline CI-CD/AzureDevOps/Mend CLI/AzureDevOps-windows.yaml:33]: URL should be the Windows link

File: CI-CD/AzureDevOps/Mend CLI/AzureDevOps-windows.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+		
+    ### Run SCA scan ###
+    echo Start Mend SCA scan
+    mend sca -u
```
Comment (tabacmend) [inline CI-CD/AzureDevOps/Mend CLI/AzureDevOps-windows.yaml:37]: As this is powershell, I think we should add ./mend.exe for SCA and SAST

File: CI-CD/AzureDevOps/Mend CLI/AzureDevOps-windows.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    
+    ### Run SAST scan ###
+    echo Start Mend SAST scan
+    mend sast --name=$(Build.Repository.Name)_$(Build.SourceBranchName) --app AZ$(System.TeamProject)_$(Build.Repository.Name)
```
Comment (tabacmend) [inline CI-CD/AzureDevOps/Mend CLI/AzureDevOps-windows.yaml:41]: Same here ./mend.exe

**Review by tabacmend — CHANGES_REQUESTED** [review]

File: CI-CD/AzureDevOps/Mend CLI/AzureDevOps-windows.yaml (hunk 1 above)
Comment (lukebrogan-mend) [reply CI-CD/AzureDevOps/Mend CLI/AzureDevOps-windows.yaml]: fixed, thanks!

**Review by tabacmend — APPROVED** [review]

**Pull request merged**